pkg/services/vimmachine.go

```diff
@@ -47,8 +47,31 @@ type VimMachineService struct {
 	Client client.Client
 }
 
+// GetMachinesInCluster returns a list of VSphereMachine objects belonging to the cluster.
+func (v *VimMachineService) GetMachinesInCluster(
```

Comment:
Makes sense, since the client is a part of the `VimMachineService` and `VMopMachineService`, no use passing them in the function call. Missed that since most of it was copying over the function as is from the util package.